compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/parser/BLangNodeTransformer.java

```diff
@@ -559,20 +559,25 @@ public BLangNode transform(ModulePartNode modulePart) {
 
     @Override
     public BLangNode transform(ModuleVariableDeclarationNode modVarDeclrNode) {
+
         TypedBindingPatternNode typedBindingPattern = modVarDeclrNode.typedBindingPattern();
+
         BindingPatternNode bindingPatternNode = typedBindingPattern.bindingPattern();
+        Optional<io.ballerina.compiler.syntax.tree.ExpressionNode> initializer = modVarDeclrNode.initializer();
+
+        //Get target variable
```

Comment:
Shall we revisit the comments and remove the ones that are not required?

Reply:
Sure